addons/html_builder/static/src/utils/grid_layout_utils.js

```diff
@@ -206,6 +206,7 @@ function placeColumns(columnEls, rowSize, rowGap, columnSize, columnGap) {
     const columnSpans = [];
     let zIndex = 1;
     const imageColumns = []; // array of boolean telling if it is a column with only an image.
+    const isRtl = !!columnEls[0]?.closest(".o_rtl, [dir='rtl']");
```

Comment:
If you do `!!columnEls[0]`, you transform it as a boolean, so doing `closest` is not correct => `columnEls[0]?...` is enough, like you did previously :+1: 

EDIT: in fact it does not, but I find the notation confusing

Reply:
Yeah I was not talking about no doing `!!` its the whole line combining `!!` and `?` that I find confusing, of course you should make it a boolean :+1: 
But in that case I would have preferred: `!!columnEls[0] && !!columnEls[0].closest(...)`
But anyway, it's a nitpick :+1: